checks/http/roundcube.rb

```diff
@@ -0,0 +1,42 @@
+module Intrigue
+module Ident
+module Check
+class Roundcube < Intrigue::Ident::Check::Base
+
+    def generate_checks(url)
+    [
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["Email"],
+            :vendor => "Roundcube",
+            :product => "Roundcube",
+            :website => "https://roundcube.net/",
+            :match_details => "Roundcube title",
+            :version => nil,
+            :match_type => :content_body,
+            :match_content =>  /:: Welcome to Roundcube/i,
```

Comment:
can this be tightened up ? Also if this is matching title only, use content_title (see README)

Reply:
Usually the title goes like so:
"Roundcube Webmail :: Welcome to Roundcube Webmail"
 But i have found the rare occasion where the text before  ":: Welcome to Roundcube" was not always the same, I can tighten it to say that it ends with :: Welcome to Roundcube Webmail

Reply:
got it okay - perfectly okay leave it open if it's catching multiple cases. Just try to catch it from the beginning or end. Esp in cases where it's a body regex. (since it can catch all sorts of stuff). 

I would suggest something like: 
```match_type => :content_title, 
match_content => /^Roundcube Webmail ::.*$/
```